### PR TITLE
Homepage Discussion

### PR DESCRIPTION
**Pull request by bcbrennecke**
This pull request is for the purposes of discussion for now.
The gif only records at 4fps, the actual loading of the wordcloud is actually very responsive.  I'm attaching a regular screenshot too.
This is a dynamic word cloud, it generated every time the user hits refresh or when the page is resized.
Getting Started is going to link the the FAQ cards (Which I think i'm changing to something else)
All Ideas will link to all the ideas showing up on the page whether its on my page or the ideas page

wordcloudanimate
wordcloudresize
image
Discuss: @YashdalfTheGray @Mctalian @alanlgirard @davethenipper @LongArmMcGee @monotkate

### CODE REVIEW

**Comment by davethenipper** [comment]
Discussion:  We love the word cloud!

File: src/app.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ angular.module('flintAndSteel', [
         'ui.router',
         'ngMaterial',
         'ngMessages',
-        'ui.identicon'
+        'ui.identicon',
```
Comment (YashdalfTheGray) [inline src/app.js]: This comma isn't needed.

File: src/homeView/homeView.css [hunk 2]
```diff
@@ -462,4 +467,94 @@ body {
...
+div#wordcloud a:hover {
+  color: #0df;
+}
+div#wordcloud a:hover {
+  color: #0cf;
+}
+div#wordcloud span {
+  padding: 0;
+}
+div#wordcloud span.w10 {
```
Comment (YashdalfTheGray) [inline src/homeView/homeView.css]: Last time I checked you weren't using these `div#wordcloud span.wxx` classes. If you're not using them, get rid of them.

File: src/homeView/homeView.css [hunk 3]
```diff
@@ -462,4 +467,94 @@ body {
...
+    padding: 0.5em;
+    text-shadow: 0px 1px 1px #fff;
+    border-top: 1px dotted #ddd;
+}
+#footer a {
+    color: #0af;
+    text-decoration: none;
+}
+
+div#wordcloud {
```
Comment (YashdalfTheGray) [inline src/homeView/homeView.css]: Tabbing needs to be fixed for the next few classes.

File: src/homeView/homeView.css [hunk 4]
```diff
@@ -1,13 +1,17 @@
-.homeview-image {
```
Comment (YashdalfTheGray) [inline src/homeView/homeView.css]: All these commented out classes shouldn't be checked in.

**Comment by YashdalfTheGray** [comment]
`homeView.js` and `homeView.tpl.html` stil need cleanup. 
* Links when you click on hyperlinked words should go to something meaningful
* That large array of lorem ipsum type words should not be there
* There is code that is commented out in the template that should be deleted.

**Comment by bcbrennecke** [comment]
@YashdalfTheGray I didn't see you were auditing this until now.  I just put this up to review the cloud itself, nothing else is done here.  Everyone in the meeting just wanted to see what it looked like to talk about.

**Comment by bcbrennecke** [comment]
@YashdalfTheGray No code audit necessary.  I have things to change.

**Comment by YashdalfTheGray** [comment]
Oh okay, sorry about that.

Yash Kulshrestha
yash.kulshrestha@gmail.com
(614)563-1163

**Pull request closed without merging**